_locales/da/messages.json

```diff
@@ -87,6 +87,26 @@
     "message":"Forøg størrelsen af forstørrede billeder med denne mændge",
     "description":"[options] Tooltip for adjust the size of zoomed images option"
   },
+  "optCaptionLocation": {
```

Comment:
please remove messages from all locales except for "en" - it's used as a default fallback locale